src/app/chatbot-page/interview/answers.service.ts

```diff
@@ -1,30 +1,35 @@
-// answers.service.ts
 import { Injectable } from '@angular/core';
 
 @Injectable({
   providedIn: 'root'
 })
+
 export class AnswersService {
 
   private answerdata: any[] = [];
 
-
+//Stores the answer for a question;
   saveAnswer(questionNumber: any, answer: any) {
-    // Suche die Frage in den Daten
+
+    // Search for the question in the data;
     let questionData = this.answerdata.find(q => q.question === questionNumber);
-    // Wenn die Frage nicht gefunden wurde, erstelle eine neue Frage
+
+    // If the question was not found, create a new question;
     if (!questionData) {
       questionData = { question: questionNumber, answers: [] };
       this.answerdata.push(questionData);
     }
-    // Füge die Antwort zur Frage hinzu
+    // Add the answer to the question;
     questionData.answers.push(answer);
+
     console.log(this.answerdata);
```

Comment:
We shouldn't need to log any data to the console, if it was only for testing, then delete this line